lib/cart_point.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:math';
+
+class Cart_Point {
```

Comment:
https://www.dartlang.org/guides/language/effective-dart/style I think UpperCamelCase for Cart_Point